README.md

```diff
@@ -46,6 +46,7 @@ $app->add(new \Slim\Middleware\Session([
   is made (interaction with server).
 * `handler`: Custom session handler class or object. Must implement
   `SessionHandlerInterface` as required by PHP.
+* `ini_settings`: array of php.ini directives [related to the sessions](http://php.net/manual/en/session.configuration.php) (as keys) with overriding values (for ex: `['session.gc_maxlifetime' => 86400]`)
```

Comment:
As this patch is changing the operation of the package, perhaps the docs here should have the old defaults, i.e. `['session.gc_probability' => 1, 'session.gc_divisor' => 1, 'session.gc_maxlifetime' => 30 * 24 * 60 * 60 ]`

(And does this mean the major version number should be incremented?)

Reply:
Added note to docs

Despite operation of the package is changed, this changes will not broke existing applications, but may prevent other people to run into same troubles. So, I think incrementing of minor version should be enough.